File: src/main/scala-graphql/io/github/graphql/model/PullRequestReviewCommentStateTO.scala
package io.github.graphql.model

import com.fasterxml.jackson.core.`type`.TypeReference

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:00+0800"
)
object PullRequestReviewCommentStateTO extends Enumeration {

    type PullRequestReviewCommentStateTO = Value

    val PENDING: Value = Value("PENDING")
    val SUBMITTED: Value = Value("SUBMITTED")

}

class PullRequestReviewCommentStateTOTypeRefer extends TypeReference[PullRequestReviewCommentStateTO.type]

File: src/main/java-graphql/io/github/graphql/model/PullRequestReviewCommentStateTO.java
package io.github.graphql.model;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:47+0800"
)
public enum PullRequestReviewCommentStateTO {

    PENDING("PENDING"),
    SUBMITTED("SUBMITTED");

    private final String graphqlName;

    private PullRequestReviewCommentStateTO(String graphqlName) {
        this.graphqlName = graphqlName;
    }

    @Override
    public String toString() {
        return this.graphqlName;
    }

}
